src/lib/orionTypes/OrionValueType.h

```diff
@@ -34,16 +34,23 @@ namespace orion
 /* ****************************************************************************
 *
 * ValueType - 
+*
+* Note ValueTypeNotGiven is only used by NGSIv1 parsing logic. In NGSIv2 parsing,
```

Comment:
Note: ...
   or
Note that ...

Reply:
Fixed in e85892cdb28f2f510fd2b0a38cacbb43082f9613